### PR TITLE
Homework4: Maria Lichko

### PR DESCRIPTION
**Pull request by maryiaLichko**
The homework #4 done without the exercise 6

### CODE REVIEW

File: src/dictCompr [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Создайте словарь с помощью генератора словарей,
```
Comment (belovmd) [inline src/dictCompr]: why this file doesn't have .py extension?

File: src/exercise3.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Даны два списка чисел. Посчитайте, сколько
```
Comment (belovmd) [inline src/exercise3.py:1]: it's not correct solution. it should be symmetrical difference

File: src/exercise6.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Во входной строке записан текст. Словом считается
+# последовательность непробельных символов идущих подряд,
+# слова разделены одним или большим числом пробелов или
+# символами конца строки. Определите, сколько различных
+# слов содержится в этом тексте.
+
```
Comment (belovmd) [inline src/exercise6.py:6]: try to use only split. this task has one-line solution

**Review by belovmd — CHANGES_REQUESTED** [review]
1. Please name PR according to our guidelines. 
2. missed task №5
3. Please address next issues:

File: src/exercise6.py (hunk 3 above)
Comment (maryiaLichko) [reply src/exercise6.py]: thank you for the review. changes were done

Comment (maryiaLichko) [reply src/exercise6.py:6]: > try to use only split. this task has one-line solution

thank you for the review. changes were done

File: src/exercise3.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+# чисел входит только в один из этих списков
+
+m = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+n = [10, 11, 12, 13, 4, 15, 16, 17, 7]
+
+print(len(set.intersection(set(m), set(n))))
```
Comment (belovmd) [inline src/exercise3.py:7]: symmetric difference, not intersection

File: src/exercise2_cities.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+count_countr = int(input('Please, enter the number of countries: '))
+for _ in range(count_countr):
+    country, *cit = input().split()
+    countr[country] = cit
+
+count_cit = int(input('Please, enter the number of cities: '))
+
+for _ in range(count_cit):
+    city = input()
+    for country, cit in countr.items():
```
Comment (belovmd) [inline src/exercise2_cities.py:41]: o(n*m) complexity revert this dict is a good idea

**Review by belovmd — COMMENTED** [review]
missed languages task. 
6,5

**Review by belovmd — COMMENTED** [review]
would be re-reviewed 3/22

**Review by belovmd — COMMENTED** [review]
6,5

**Pull request closed without merging**